src/app/pages/Map/Map.tsx

```diff
@@ -243,13 +238,77 @@ export default function MapBox(): JSX.Element {
       .coordinates as LngLatLike
   }
 
+  //set marker
   map.current && midpoint
     ? new mapboxgl.Marker({ color: '#2b5113' })
         .setLngLat(midpoint)
         .addTo(map.current)
     : null
+  const middle: number[] = midpoint as number[]
+
+  //get POI / hotels und  Unterkünfte 25 stk
+  async function getPOI(
```

Comment:
This file already seems pretty long. Do you have some functions like this maybe that you could outsource?

Reply:
I have this on my list, but since its applying the layer to the map, i dont find an easy way to outsource these functions :) but you are right, it needs to be shorter!